Packs/MicrosoftDefenderThreatIntelligence/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "MicrosoftDefenderThreatIntelligence",
+    "description": "MicrosoftDefenderThreatIntelligence",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "marketplaces": [
```

Comment:
Add platform to the marketplaces with the supported modules as follow -    
 "supportedModules": [
        "X1",
        "X3",
        "X5",
        "ENT_PLUS",
        "agentix"